src/GUI/FunctionControlPanel.java

```diff
@@ -62,6 +65,16 @@ public void actionPerformed(ActionEvent e) {
             removeFunctionControlPanel(FunctionControlPanel.this);
         }
     }
+    private class TableActionListener implements ActionListener {
+        @Override
+        public void actionPerformed(ActionEvent e) {
+            if (instanceID != -1) {
+                ParametricFunction function = graphPanel.getFunction(instanceID);
+                Valuetable valuetable = new Valuetable(function);
```

Comment:
Variable valuetable is never used anywhere else. So new Valuetable would be sufficient.